examples/nuclei_image_segmentation/README.md

```diff
@@ -0,0 +1,13 @@
+# Nuclei Image Segmentation With UNets
+
+## Setup
+
+1. Install dependencies: `pip install -r requirements.txt`
```

Comment:
> I think it could be beneficial if we formulate what are the points that this notebook is supposed to teach.

From my reading, this notebook is intended for users who are familiar with the general topic (image segmentation) and pytorch but want to get into skorch. I agree that this could be made explicit at the start. However, I find the amount of code shown in the notebook to be sufficient. The reader may or may not look at the particular code if they like. Since the code is sufficiently clear, this should be no problem (a *short* docstring for some of the classes and functions would be nice, though).

> This is step is somewhat strange since it will only get the dependencies of skorch, not skorch itself or Jupyter.

The install instruction in the notebook is better, true (but it also lacks the instruction to install jupyter notebook). Either improve on this instruction or leave it out since it's already documented. What I would like to see in the README is a link to the notebook using nbviewer. Then the user can look at the installation instruction in the notebook before needing to isntall jupyter.

Reply:
> From my reading, this notebook is intended for users who are familiar with the general topic (image segmentation) and pytorch but want to get into skorch

That's a legit target. But then perhaps more emphasis can be put where skorch features are used. Right now it feels more like a "how to solve that dataset" example, but should be more like "how skorch makes it easy to solve that dataset".